Applications/PaintBrush/EraseTool.h

```diff
@@ -15,6 +15,7 @@ class EraseTool final : public Tool {
     virtual void on_contextmenu(GContextMenuEvent&) override;
 
 private:
+    Color get_color();
```

Comment:
This should be const since it does not mutate the `EraseTool` state:
```suggestion
        Color get_color() const;
```